addons/account/models/account_payment.py

```diff
@@ -819,6 +823,7 @@ def _create_paired_internal_transfer_payment(self):
                 'payment_type': payment.payment_type == 'outbound' and 'inbound' or 'outbound',
                 'move_id': None,
                 'ref': payment.ref,
+                'date': payment.date,
```

Comment:
@smetl Can you check this if we can do this as a generic fix?

Reply:
Idk if this is intended or not :/

Reply:
without this, if you create a transfer for a date different from today, the paired transfer will be created for today. IMHO, that would not be the expected behavior, with this change, both payments (of type transfer) are created for on the same date